### ISSUE TITLE
Webpack resolver: possible compatibility issue with atom/linter-eslint

### ISSUE THREAD

**Issue by Moeriki** [benmosher/eslint-plugin-import#262]
Dear,

I've been having a compatibility issue with the webpack resolver and atom/linter-eslint. When running eslint from the command line all my imports gets resolved correctly. Through atom/linter-eslint it could not resolve those that relied on `resolve.root` or `resolve.alias`.

I'm not sure if this is a bug in either this plugin, linter-eslint, or a different depending package.

A work-around is simple enough so I just wanted to highlight how to fix it for the benefit of others. Perhaps this could be included in the documentation as well.

My webpack config before: **not working for linter-eslint**

```javascript
{
  alias: {
    'app': path.resolve('./client/app'),
  }
}
```

My webpack config after: **working for linter-eslint**.

```javascript
{
  alias: {
    'app': path.resolve(__dirname, './client/app'),
  }
}
```

Same goes for the `resolve.root` setting.

Cheers.

**Comment by benmosher** [comment]
You're in good company: #260

**TL;DR:** your described workaround is the preferred solution at this time. See the linked issue for possible alternatives, but I'm not sure what can be done in the resolver to solve this issue.
